eng/helix/content/runtests.sh

```diff
@@ -95,6 +98,17 @@ if [ -d "Microsoft.AspNetCore.App" ]
 then
     echo "Found Microsoft.AspNetCore.App directory, copying to $DOTNET_ROOT/shared/Microsoft.AspNetCore.App/$dotnet_runtime_version."
     cp -r Microsoft.AspNetCore.App $DOTNET_ROOT/shared/Microsoft.AspNetCore.App/$dotnet_runtime_version
+    
+    echo "Adding current directory to nuget sources: $DIR"
+    dotnet nuget add source $DIR
+    dotnet nuget add source https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet5/nuget/v3/index.json
+    dotnet nuget list source
+    
+    dotnet tool install dotnet-ef --global --version $efVersion
+    
+    # Ensure tools are on on PATH
+    export PATH="$PATH:$DOTNET_CLI_HOME/.dotnet/tools"
```

Comment:
Why does this directory need to be on the path? Won't `dotnet` find it automatically?

Reply:
Nope, probably because we install the CLI in this script